c++/src/kj/cidr.c++

```diff
@@ -41,6 +41,10 @@
 #include <arpa/inet.h>
 #endif
 
+#if __FreeBSD__
+#include <netinet/in.h>
```

Comment:
Is this really specific to FreeBSD, or should we really be including this on all systems? (Or at least all non-Windows systems.)

Reply:
The reason I included it is because the build fails without it. If the build succeeds on all other non-Windows systems, I see no reason to include it (in other words, "if it ain't broke, don't fix it").